src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -1,22 +1,400 @@
 /* eslint-disable react/jsx-props-no-spreading */
+import {
+  Box,
+  Center,
+  Icon,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Thead,
+  Tr,
+  Text,
+  TextProps,
+  Skeleton,
+} from "@chakra-ui/react"
+import { Button, Checkbox, Link } from "@opengovsg/design-system-react"
+import { useForm } from "react-hook-form"
 
 import { useSiteLaunchContext } from "contexts/SiteLaunchContext"
 
+import { BxCopy, BxLifeBuoy } from "assets"
+import {
+  DNSRecord,
+  SITE_LAUNCH_PAGES,
+  SITE_LAUNCH_TASKS,
+  SITE_LAUNCH_TASKS_LENGTH,
+} from "types/siteLaunch"
+
 import { SiteLaunchPadBody } from "./SiteLaunchPadBody"
 import { SiteLaunchPadTitle } from "./SiteLaunchPadTitle"
 
+const TITLE_TEXTS = [
```

Comment:
note to reviewers, I cannot represent the subtitles in the same manner as 
1) not all tasks have subtitles 
2) not all subtitles are rendered as a `<Text/>` element. (well they are but one got a nested <Link>, so this layer of abstraction not suitable for subtitles